mscore/scoreview.cpp

```diff
@@ -3732,8 +3732,10 @@ void ScoreView::cloneElement(Element* e)
             e = toSpannerSegment(e)->spanner();
       mimeData->setData(mimeSymbolFormat, e->mimeData(QPointF()));
       drag->setMimeData(mimeData);
-      drag->setPixmap(QPixmap());
-      drag->start(Qt::CopyAction);
+      static QPixmap pixmap = QPixmap(2, 2);    // null or 1x1 crashes on Linux under ChromeOS?!
```

Comment:
Could you also test whether crash happens if not setting any pixmap at all (that is, if this `setPixmap()` call is just removed)? Maybe we should just not set any pixmap if we don't need it. I don't get though why setting it should break something but this crash overall seems to be something mysterious.

Reply:
I did try that, and a dozen or so other variations, no luck.  The only thing that seems to work is setting a pixmap at least 2z2.  From reading through the documentation, I see there is mention of Qt trying to align the pixmap with the cursor there was talk of centering, making me wonder if somewhere there is a division by zero if the pixmap isn't at least two pixels wide.  But whatever the reason, the crash is totally reproducible without the change, works perfectly with the change.

I could put this in an #ifdef LINUX or whatever makes the msot sense if we're concerned, but it seems harmless in any case.  I also considered giving it a real icon to show, like our "Copy"" icon, but decided to leave it at this.